js/util/test-head.js

```diff
@@ -17,12 +13,13 @@ const _createStore = configureStore({
   }
 });
 
-function createStore() {
+function createStore(threadClient) {
+  mockThreadClient = threadClient;
```

Comment:
Why not move `configureStore` into here and dynamically create a new store constructor each time and just close over the mock thread client?